go/libkb/test_common.go

```diff
@@ -116,10 +116,10 @@ func (tc *TestContext) Cleanup() {
 		tc.G.Log.Warning("tc.G.Shutdown failed: %s", err)
 	}
 	if len(tc.Tp.Home) > 0 {
-		tc.G.Log.Debug("cleaning up %s", tc.Tp.Home)
-		os.RemoveAll(tc.Tp.Home)
 		tc.G.Log.Debug("clearing stored secrets:")
 		err := tc.ClearAllStoredSecrets()
+		tc.G.Log.Debug("cleaning up %s", tc.Tp.Home)
+		os.RemoveAll(tc.Tp.Home)
```

Comment:
Linux split store stores information in both file-based and keychain, so it needs to clear secrets before deleting home directory so it knows what secret to delete from keychain.